sw/device/lib/crypto/drivers/entropy.c

```diff
@@ -328,13 +329,41 @@ static status_t csrng_send_app_cmd(uint32_t reg_address,
   }
 
   HARDENED_TRY(csrng_fsm_idle_wait());
-
+  uint32_t cmd_reg;
+  uint32_t sts_reg;
+  uint32_t rdy_bit;
```

Comment:
nit: I'd rename these slightly so that it's clear they hold register addresses, not register contents; this will better match naming elsewhere in the cryptolib drivers.
```suggestion
  uint32_t cmd_reg_addr;
  uint32_t sts_reg_addr;
  uint32_t rdy_bit_addr;
```

Reply:
Good point, thanks!